template/common.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
```

Comment:
Can we rename this to index.html? In order to keep consistency with the dist/index.html generated file.